main.go

```diff
@@ -141,6 +141,15 @@ func handleRepo(ctx context.Context, client *github.Client, repo *github.Reposit
 	opt := &github.ListOptions{
 		PerPage: 100,
 	}
+
+	teams, resp, err := client.Repositories.ListTeams(ctx, *repo.Owner.Login, *repo.Name, opt)
+	if resp.StatusCode == http.StatusNotFound || resp.StatusCode == http.StatusForbidden {
```

Comment:
Is this better to do as a greater-then check instead?

`if resp.StatusCode >= 400`

Reply:
Your suggestion makes more sense, as it covers all potential HTTP errors, I just replicated the error checks currently implemented for all other entities (like _Collaborators_, _Keys_, _Hooks_, etc)

Reply:
Let's make things simple for upstream PRs, and do this PR with the current error handling, and then file a different PR to clean up the error handling everywhere?

Reply:
Sounds good, will do that now!